cmd/aries-js-worker/index.html

```diff
@@ -13,22 +13,39 @@
     <meta charset="utf-8"/>
     <script src="dist/web/aries.js"></script>
     <script>
-        // TODO fix the stutter in the API's exports ("Aries.Aries....")
-        const aries = Aries.Aries({assetsPath: "/dist/assets"})
+        var aries
         var stopNotifier
 
-        async function handleStart() {
+        async function startAries() {
             const request = document.querySelector("#opts").value
-            const response = await aries.start(request)
-            document.querySelector("#output").value = JSON.stringify(response)
+
+            // TODO fix the stutter in the API's exports ("Aries.Aries....")
+            aries = Aries.Aries({assetsPath: "/dist/assets", startOpts: request})
+
+            //TODO wait for event/callback to confirm if aries is started
+            document.querySelector("#output").value = "aries started"
         }
 
-        async function handleStop() {
-            const response = await aries.stop()
+        async function destroyAries() {
```

Comment:
there is not currently an await in this func

Reply:
fixed it